### DISCUSSION TITLE
How to change the directory of where the models are?

### DISCUSSION THREAD

**Discussion by AlgorithmicKing737** [lllyasviel/Fooocus#3114] [Q&A] [answered]
I use ComfyUI and all my models are downloaded in my ComfyUI/models/checkpoints folder so how do i make it so that Fooocus uses the models from there so that I don't have to duplicate those models.

**Comment by mashb1t** [comment, marked as answer]
https://github.com/lllyasviel/Fooocus/issues/3111#issuecomment-2153472503, you can modify your config.txt file. Please find all config options in the config example file.

**Reply by AlgorithmicKing737** [reply]
wow thanks a lot that was fast
